src/social-web/routes/feed/stage.tsx

```diff
@@ -125,18 +131,44 @@ export default function FeedStage( { onSelectItem }: FeedStageProps ) {
 		[ view.fields, updateView ]
 	);
 
+	// Extract ap_object_type filter from view.filters
+	const apObjectTypeFilter = useMemo( () => {
+		const typeFilter = view.filters?.find( ( f ) => f.field === 'ap_object_type' );
+		return typeFilter?.value as number[] | undefined;
+	}, [ view.filters ] );
+
 	const { feed, isResolving, totalItems, totalPages } = useFeed( {
 		perPage: view.perPage || 20,
 		page: view.page || 1,
 		orderBy: view.sort?.field || 'date',
 		order: view.sort?.direction || 'desc',
 		search: view.search || '',
 		userId: activeActorId,
+		apObjectType: apObjectTypeFilter,
 	} );
 
+	// Create ap_object_type filter field
+	const apObjectTypeField: Field< FeedPost > = useMemo(
+		() => ( {
+			id: 'ap_object_type',
+			label: __( 'Type', 'activitypub' ),
+			enableHiding: false,
+			enableSorting: false,
+			elements:
+				apObjectTypes?.map( ( term: any ) => ( {
+					value: term.id,
+					label: term.name,
+				} ) ) || [],
+			filterBy: {
+				operators: [ 'isAny' ],
```

Comment:
```suggestion
				operators: [ 'is' ],
```

We'd probably only want one to be selected at any time?

Reply:
I am not sure about that, but let's start with one

Reply:
It does have to be isAny... a taxonomy seem to not work otherwise:

> 'is' operator:
> - Used for single-value comparisons
>  - Matches when the field value exactly equals the filter value
>  - Best for boolean or single-select filters
>
>  'isAny' operator:
>  - Used for multi-value/enum comparisons
>  - Matches when the field value is any one of the selected filter values
>  - Required when you have elements defined (multiple options to choose from)
>  - Allows users to select multiple types (e.g., filter for "Article OR Note")
>
>  In your case, since you defined elements with multiple object types (Article, Note, etc.) and users can potentially select multiple types to filter by, WordPress DataViews expects the 'isAny' operator to
  properly:
>
>  1. Display the filter UI with checkboxes for multiple selections
>  2. Apply the filter logic correctly (show posts that match ANY of the selected types)
>  3. Store and persist the filter state properly
>
>  With 'is', DataViews likely didn't recognize it as a valid enum-style filter and either didn't display the filter UI at all or didn't persist the selection correctly. The 'isAny' operator tells DataViews: "this
>   field has predefined options (elements) and we want to filter by any matching value."